src/hooks/useSetUserProfileEffect.ts

```diff
@@ -4,7 +4,7 @@ import { useUpdateUser, useUser } from './useUser';
 
 export const useSetUserProfileEffect = () => {
   const { isLoading, isFetched, data: user } = useUser();
-  const { mutate: updateUser } = useUpdateUser();
```

Comment:
I think this was probably better before since we are actually updating the user's data here.

Reply:
Ah, yeah, find all found too many 😮‍💨